articles/virtual-machines/workloads/redhat/redhat-images.md

```diff
@@ -193,6 +193,15 @@ We update only the current minor release in a given image family. With the relea
 
 Current policy is to keep all previously published images. We reserve the right to remove images that are known to cause problems of any kind. For example, images with incorrect configurations due to subsequent platform or component updates might be removed. Images that might be removed follow the current Azure Marketplace policy to provide notifications up to 30 days before image removal.
 
+## Network configuration
+
+Current RHEL 8 and RHEL 9 images leverage the network configuration of DHCP on eth0. The default cloud-init network configuration is overridden by setting "apply_network_config: false".
```

Comment:
can we reword it to 
"we are currently overriding the default cloud-init network configuration by setting "apply_network_config: false" to support RHEL 8 and RHEL 9 images and customers."